lib/core.h

```diff
@@ -134,7 +134,7 @@ void secure_wipe_memory(void *v, size_t n);
  * @param mem Pointer to the memory
  * @param s Memory size in bytes
  */
-void clear_internal_memory(void *v, size_t n);
+ARGON2_API void clear_internal_memory(void *v, size_t n);
```

Comment:
Why do you want to export `clear_internal_memory`? AFAIK it should be only used by the library code itself.

Reply:
From my perspective it looks like this. There are 3 projects: `argon2`, `argon2-exec` and `argon2-test`. The first one defines `clear_internal_memory`, two others make use of it in `test.c` and `run.c`.  Hence, there is a link error without export.

Reply:
Oh, right, those two use it, too... The encapsulation in the original code is just not very good :/ I guess for now it is best to keep it simple and just mark `clear_internal_memory` with the exisitng `ARGON2_PUBLIC` (if that is enough).